Mu2eKinKal/test/TTKK.fcl

```diff
@@ -0,0 +1,140 @@
+# Minimal KinKal job that fits digis to a downstream electron as a trigger seed fit , and analyzes the output with TrkAna
+# To create a functional job you must add database purpose and version; ie create a fcl stub that #includes this file and
+# adds (for instance):
+# services.DbService.purpose: MDC2020_perfect
+# services.DbService.version: v1_0
+#
+#include "Offline/fcl/minimalMessageService.fcl"
+#include "Offline/fcl/standardServices.fcl"
+#include "Production/JobConfig/reco/prolog.fcl"
+#include "Offline/Mu2eKinKal/fcl/prolog_trigger.fcl"
+#include "Offline/TrkHitReco/fcl/prolog_trigger.fcl"
+#include "Offline/TrkReco/fcl/prolog_trigger.fcl"
+#include "Offline/TrkPatRec/fcl/prolog_trigger.fcl"
+#include "Offline/TrkFilters/fcl/prolog_trigger.fcl"
+#include "Offline/CalPatRec/fcl/prolog.fcl"
+#include "Offline/CalPatRec/fcl/prolog_trigger.fcl"
+#include "Offline/Trigger/fcl/prolog_trigger.fcl"
```

Comment:
not relevant for the approval, but `Trigger/fcl/prolog_trigger.fcl` includes all the prolog_triggers.fcl listed above and it encapsulates them in the `Trigger.producers(filters)` blocks

Reply:
good point.  I'll clean that up.